server/src/data-processors/helpers/extract-token.js

```diff
@@ -0,0 +1,22 @@
+const getType = require('../../global-helpers/get-type');
+const throwError = require('../../global-helpers/throw-error');
```

Comment:
Just a suggestion: might it be easier to export all global helpers from an `index.js` file, thereby allowing you to import the destructured functions? Could potentially make for cleaner imports.

Reply:
Maybe, yeah that's a good idea. 🤔

Reply:
I'm full of those. Might save you a bit of a headache later down the line.